python/pyarrow/_substrait.pyx

```diff
@@ -39,7 +39,7 @@ cdef CDeclaration _create_named_table_provider(dict named_args, const std_vector
         c_name = names[i]
         py_names.append(frombytes(c_name))
 
-    py_table = named_args["provider"](py_names)
+        py_table = named_args["provider"](py_names)
```

Comment:
Not sure why this is indented? Why should it run within the loop?

Reply:
Oops, good catch!  I was experimenting with this function for something unrelated and didn't notice the change because I was hiding whitespace changes when doing self-review.

Thanks, I have reverted this.